src/applications/personalization/profile/tests/components/direct-deposit/BankInfo.unit.spec.jsx

```diff
@@ -110,9 +110,18 @@ describe('DirectDepositCNP', () => {
     server.close();
   });
 
+  let bankTypeBeingEdited = null;
+  const setBankTypeBeingEdited = bankType => {
```

Comment:
I don't think you need to fully mock this out since these unit tests never test anything that _really_ use the functionality of the prop this function is passed in as...